README.md

```diff
@@ -4,33 +4,48 @@ Singleton factory used by Safe related contracts based on https://github.com/Ara
 
 The original library used a presigned transaction without a chain id to allow deployment on different chains. Some chains do not allow such transactions to be submitted (e.g. Celo and Avalanche) therefore this repository will provide the same factory that can be deployed via a presigned transaction that includes the chain id. The key that is used to sign is controlled by the Safe team.
 
-# Adding new networks
+# User documentation
 
-To add support for new networks the same key used for the existing networks should be used to generate a presigned transaction for a new network. To request support for a new network please open a [new issue](https://github.com/safe-global/safe-singleton-factory/issues/new/choose).
+## How to get singleton deployed for you network
+
+As the singleton is EIP155 protected, we need to sign the singleton for your network. But some requisites must be met before we do that, the most important one is having funds on the deployer so we can deploy the contract.
```

Comment:
```suggestion
As the singleton is deployed with an EIP155 transaction, we need to sign the deployment transaction for your network. But some prerequisites must be met before that, and the most important one is having funds on the deployer so we can deploy the contract.
```
I think it's more correct to talk about the transaction here, not the singleton itself

Reply:
Sure